README.md

```diff
@@ -1,6 +1,20 @@
  ## Virtual Pet
-A Javascript virtual pet. 
+A Javascript virtual pet project, created as part of the Manchester Codes Javascript module. 
 
 ## Features 
 
+* You can name your pet:
+`const pet = new Pet("Gerald")`
+
+* Feed your pet:
+`pet.feed()`
+
+* Walk your pet:
+`pet.walk()`
+
+ * Give your pet a check up:
+`pet.checkUp()`
+
+
```

Comment:
I would, later on, once you have some free time, expand on this README. You can explain the approach and logic behind it! Companies tend to look at your github projects when you're looking for a first job.